src/index.js

```diff
@@ -10,7 +10,7 @@ function rfdc (opts) {
   function cloneArray (a, fn) {
     const keys = Object.keys(a)
     const a2 = new Array(keys.length)
-    for (var i = 0; i < keys.length; i++) {
+    for (let i = 0; i < keys.length; i++) {
```

Comment:
It is intentional to use `var` here due to supporting older Node.js versions which did not handle that well.